exonum/src/storage/db.rs

```diff
@@ -468,11 +468,15 @@ impl Fork {
         } else {
             changes.data.clear();
         }
-        // Remove from storage
-        let mut iter = self
-            .snapshot
-            .iter(name, prefix.map_or(&[], |k| k.as_slice()));
+
+        // Remove keys from storage.
+        let prefix_as_slice = prefix.map_or(&[] as &[u8], |k| k.as_slice());
```

Comment:
What about this syntax?
```suggestion
        let prefix_as_slice: &[u8] = prefix.map_or(&[], |k| k.as_slice());
```

By the way, why we use `&Vec` in the arguments? If we replace it with `&[u8]` (which is better for API) - we can use even:

```suggestion
        let prefix_as_slice = prefix.unwrap_or_default();
```